### ISSUE TITLE
:on is not an editor command

### ISSUE THREAD

**Issue by sw9** [VSCodeVim/Vim#3286]
**Describe the bug**
:on is not an editor command

**To Reproduce**
1.  be in normal mode
2.  type :vsp to split the screen
3.  type :on to remove the split
4.  see message :on is not an editor command

**Expected behavior**
All other splits are closed

**Environment (please complete the following information):**
- Extension Version: `0.17.0`
- OS Version: `Windows_NT x64 10.0.17134`
- VSCode version: `1.30.1`

**Comment by jpoon** [comment]
Interesting, didn't know of this command. From vim help:

```
CTRL-W CTRL-O					*CTRL-W_CTRL-O* *:on* *:only*
		Make the current window the only one on the screen.  All other
		windows are closed.  For {count} see |:quit| command.
		
		When the 'hidden' option is set, all buffers in closed windows
		become hidden.
		
		When 'hidden' is not set, and the 'autowrite' option is set,
		modified buffers are written.  Otherwise, windows that have
		buffers that are modified are not removed, unless the [!] is
		given, then they become hidden.  But modified buffers are
		never abandoned, so changes cannot get lost.
```

**Comment by aquova** [comment]
To elaborate, the `:on` command is a shortened alias for `:only`, which is properly implemented in VSCode Vim. It's simply the shortened command that isn't recognized.

**Comment by xconverge** [comment]
fixed in #3303
